bin/create-scripts/block.js

```diff
@@ -34,3 +36,44 @@ const config = {
 const compiler = webpack( config );
 
 compiler.run( ( error, stats ) => console.log( stats.toString() ) );
+
+const sassInputFile = path.resolve( sourceDir, 'style.scss' );
+if ( fs.existsSync( sassInputFile ) ) {
+	if ( ! fs.existsSync( outputDir ) ) {
+		fs.mkdirSync( outputDir );
+	}
+
+	const cssOutFile = path.resolve( outputDir, `blocks-${ blockName }.css` );
+	sass.render(
+		{
+			file: sassInputFile,
+			outFile: cssOutFile,
+			sourceMap: true,
+			outputStyle: 'compressed',
+		},
+		( error, result ) => {
+			if ( error ) {
+				console.warn( 'Failed to process SCSS', blockName, error );
+				return;
+			}
+
+			console.log( 'Rendering Complete, saving .css file...' );
+
+			fs.writeFile( cssOutFile, result.css, fileError => {
+				if ( fileError ) {
+					console.warn( 'Failed to save CSS', blockName, fileError );
+					return;
+				}
+				console.log( 'Wrote CSS to ' + cssOutFile );
+			} );
+
+			fs.writeFile( cssOutFile + '.map', result.map, fileError => {
+				if ( fileError ) {
+					console.warn( 'Failed to save sourcemap', blockName, fileError );
+					return;
+				}
+				console.log( 'Wrote Source Map to ' + cssOutFile + '.map' );
+			} );
+		}
```

Comment:
I'd like to learn how the others feel about this. I'm leery of it for a couple reasons:
 - it's adding arbitrary build steps to the existing process when actually we have a CSS process already. I'd rather take the opportunity to fix Calypso's CSS system so that it works without recreating it here. I don't mind smaller overwrites but I think this is kind of working around the problem
 - I'm wanting to stay away from magic filenames. implicit behaviors are confusing and I think it's also quite limiting. I'd rather rely on (currently) webpack's dependency graph to pull in required CSS. that is, suppose we have two blocks with two stylesheets, how do we call them? `style-1.scss` and `style-2.scss`? if we `import` them as we're wont to do with webpack then things resolve the way we expect them to and the way we have explicitly ordered them to.

if we stick to the dependency graph and explicit inclusions then we open ourselves up more long-term to on-demand loading as a by-product of improving the SDK.

Reply:
While it's true that this is adding some build steps, I was thinking that:

* We **don't** currently have a CSS process that we can share everywhere. It's a good idea to create it, of course, but we don't have it at this time, so let's start with something and then improve as we move forward. 
* We're [already doing](https://github.com/Automattic/wp-calypso/blob/master/bin/process-extensions-scss.js#L29) something similar for extensions. De-duplicating is a good idea of course, but it can be done in a subsequent PR.
* The purpose of this is to introduce a simple step to build CSS that could help us move forward. Of course we'll iterate and make it better and reduce duplications, this is only the first version. Let's get there with small steps.
* This PR is already introducing some usable changes - like bringing consistency to naming block files, passing the block directory vs the block main JS file. I'd like to land those to unblock other explorations.

What do you think?